docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -242,6 +242,30 @@ for example::
       ansible.builtin.set_fact:
         myvar: "{{ result.stdout | from_json }}"
 
+.. versionadded:: '2.12'
+
+To remove preceding and trailing unwanted lines from the string that possibly contains JSON, use the 'clean_json' filter.
+Very useful when dealing with output from applications that mix in messages with the data::
+
+    - name: Using filtered=True
+      ansible.builtin.set_fact:
+        myvar: "{{ junk_src|clean_json|from_json }}"
+      vars:
+        junk_src: |
+          first line
+          {
+            "one": "1"
+          }
+          last line
+
+will return::
+
+    {
+      "one": "1"
+    }
+
+.. note::
+    - The clean_json filter does not verify nor validate that the 'possible JSON' found is valid JSON
```

Comment:
What happens if there is no JSON in the output, and how is "no JSON" defined (no curly braces?)? What would trigger the `Unable to filter` error message?

Reply:
Yes, but also [ ], not having either will return `Unable to filter input: No start of json char found`